example/gardener-local/kind/cluster/templates/_kubeadm_config_patches.tpl

```diff
@@ -26,7 +26,7 @@
       hostPath: /etc/gardener-local/kube-apiserver/authz-config-with-seedauthorizer.yaml
     - name: authz-webhook-kubeconfig
       mountPath: /etc/gardener-local/kube-apiserver/authz-webhook-kubeconfig.yaml
-      hostPath: /etc/gardener-local/kube-apiserver/authz-webhook-kubeconfig-{{ if eq .Values.networking.ipFamily "dual" }}ipv6{{ else }}{{ .Values.networking.ipFamily }}{{ end }}.yaml
+      hostPath: /etc/gardener-local/kube-apiserver/authz-webhook-kubeconfig.yaml
```

Comment:
Why does this work for IPv6-only? I understand that the IPv4 configuration works for dual-stack, but I am confused why this should work for IPv6-only. Please explain.